app/scripts/controllers/transactions/index.js

```diff
@@ -2596,4 +2604,54 @@ export default class TransactionController extends EventEmitter {
       },
     );
   }
+
+  _requestApproval(txMeta) {
+    const id = this._getApprovalId(txMeta);
+    const { origin } = txMeta;
+    const type = MESSAGE_TYPE.TRANSACTION;
+    const requestState = { txId: txMeta.id };
```

Comment:
I'm just wondering. What's the purpose of passing `requestState` to `ApprovalController` in this instance.

I thought that was only meant for cases in which we want to update that state before the confirmation is closed.